frontend/src/metabase/visualizations/visualizations/Funnel.jsx

```diff
@@ -114,32 +114,32 @@ export default class Funnel extends Component {
     }),
     "funnel.rows": {
       section: t`Data`,
-      widget: ChartSettingOrderedRows,
+      widget: ChartSettingOrderedSimple,
       isValid: (series, settings) => {
         const funnelRows = settings["funnel.rows"];
 
         if (!funnelRows || !_.isArray(funnelRows)) {
           return false;
         }
-        if (!funnelRows.every(setting => setting.rowIndex !== undefined)) {
+        if (!funnelRows.every(setting => setting.originalIndex !== undefined)) {
           return false;
         }
 
         return (
-          funnelRows.every(setting => series[setting.rowIndex]) &&
+          funnelRows.every(setting => series[setting.originalIndex]) &&
           funnelRows.length === series.length
         );
       },
 
       getDefault: transformedSeries => {
         return transformedSeries.map(s => ({
           name: s.card.name,
-          rowIndex: s.card.rowIndex,
+          originalIndex: s.card.rowIndex,
```

Comment:
What do you think about changing other `rowIndex` references to `originalIndex` too? Since it was introduced in https://github.com/metabase/metabase/pull/25060. And having both `rowIndex` and `originalIndex` within the same file could be confusing as, correctly if I'm wrong, they refer to the same thing.

 - https://github.com/metabase/metabase/blob/1c321aa4969224e94e74944892468ceb2421a403/frontend/src/metabase/visualizations/visualizations/Funnel.jsx#L88
 - https://github.com/metabase/metabase/blob/1c321aa4969224e94e74944892468ceb2421a403/frontend/src/metabase/visualizations/visualizations/Funnel.jsx#L137
 - https://github.com/metabase/metabase/blob/1c321aa4969224e94e74944892468ceb2421a403/frontend/src/metabase/visualizations/visualizations/Funnel.jsx#L200